README.md

```diff
@@ -65,55 +65,65 @@ with:
         redeemTask: true
       dataPurge:
       formattingEvaluator:
-        scores:
-          br: 0
-          code: 1
-          p: 1
-          em: 0
-          img: 0
-          strong: 0
-          blockquote: 0
-          h1: 1
-          h2: 1
-          h3: 1
-          h4: 1
-          h5: 1
-          h6: 1
-          a: 1
-          li: 1
-          td: 1
-          hr: 0
         multipliers:
           - select: [ ISSUE_SPECIFICATION ]
```

Comment:
if I want to create a multiplier that's not specific to a particular role, how do I do that?

A cool feature would be to have like default multiplier but it can get overwritten by specific multipliers

Reply:
I suppose we can do a wildcard select first and then the later config settings can overwrite kind of like CSS behavior?
```yml
        multipliers:
          - select: [ * ]
```

Reply:
The multiplier inheritance has been an ongoing discussion yes. Before, we could create multipliers that are not tied to a role, with this new configuration it is not possible anymore. I believe the intent of the spec was to make the configuration simpler. Maybe if it is needed later we can think of it, so far the v1 never needed that much of a fine grained configuration.